src/interface/src/app/plan/create-scenarios/create-scenarios.component.html

```diff
@@ -53,7 +53,16 @@
             </div>
           </div>
         </mat-tab>
-        <mat-tab label="Outcome">Scenario Results</mat-tab>
+        <mat-tab label="Outcome">
+          <ng-container [ngSwitch]="scenarioState">
+            <app-scenario-not-started
+              *ngSwitchCase="'not-started'"></app-scenario-not-started>
```

Comment:
It may be easiest to just disable the results tab in the case where results aren't pending or completed (though I can also see the value in the instructions guiding users to finish their config first, so we can also wait for feedback to see what has the best flow)